doc-Monitoring_Alerts_and_Reporting/topics/To_schedule_a_Report.adoc

```diff
@@ -4,28 +4,39 @@
 You can view historical data by creating reports on a scheduled basis.
 In addition, scheduled reports can be emailed directly to users.
 
-. There are two ways to schedule a report.
-  Select a report from the Reports accordion and click  image:1847.png[] *Configuration*,  image:1862.png[] *Add a New Schedule*, or click the Schedules accordion and click  image:1847.png[] *Configuration*,  image:1862.png[] *Add a New Schedule*.
-+
+NOTE: You may need to disable, change the report filter, or change the frequency of a schedule.
+To do this, you will need to edit the schedule.
+
+There are two ways to schedule a report:
+
+Add a new Schedule through the *Reports* accordion:
+
+. Navigate to menu:Cloud Intel[Reports]
+. Select a report from the *Reports* accordion and click  image:1847.png[] *Configuration*.
+. Click image:1862.png[] *Add a New Schedule*. or click the Schedules accordion and click  image:1847.png[] *Configuration*,  image:1862.png[] *Add a New Schedule*.
```

Comment:
- There's a stray period that should be removed in "*Add a New Schedule*. or click"
- Also, I think the comma here should be a >  "*Configuration*,  image:1862.png[] *"